node/nanoexpress/config.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+  github: dalisoft/nanoexpress
+  version: 0.12.6
```

Comment:
can you use X.Y format ? the Z level is not very interesting / readable for the kind of person we can help

Reply:
Yea, give me 2 minute :)